README.md

```diff
@@ -117,6 +117,24 @@ The following are instructions for running our tests:
     `python setup.py install`
 4. Run the following command to test the plugin in all versions of python we support:
     `tox`
+    
+Once you have all required package installed, you can run tests locally with:
+
+```ptyhon
+
+# running all tests locally
+
+python -W always setup.py -q test
+
+# runnging an individual test
+
+python -m unittest test.[test file name].[class name].[individual test name]`
+
+# such as
```

Comment:
Perhaps `Example:` is better than `such as`.

Reply:
all updated